### ISSUE TITLE
add https://search.kiwitalk.de

### ISSUE THREAD

**Issue by keithdanielll** [searxng/searx-instances#113]
### General

- [X] It is my instance.
- [X] I give the right to check.searx.space to check my instance (every 3 hours for the response times, every 24 hours for the other tests).
- [x] I won't try to manipulate the ranking of my instance in a way that give an unfair advantage over the other public instances in the list. (e.g. caching requests for searx.space server)
- [X] I guarantee to keep an uptime per month of my instance at **minimum 90%**. Your instance will be removed if you don't respect this rule.

### Bot protection

Yes, I have installed filtron (rules.json from searx-docker).

**Comment by unixfox** [comment]
Hello @keithdanielll,

This is a friendly reminder that your provider hetzner offer IPv6, it would be great if you could add IPv6 to your instance.

It may be not difficult at all, you may already have IPv6 connectivity on your server, all you need is to add a record AAAA in your DNS server.

Here is the documentation here: https://docs.hetzner.com/cloud/servers/primary-ips/primary-ip-configuration/

The future of the internet will thank you, see [why here](https://old.reddit.com/r/explainlikeimfive/comments/1k45y7/eli5_what_is_ipv6_and_why_is_it_so_important/).
